src/date_time.rs

```diff
@@ -1 +1,30 @@
 pub use exif::DateTime as DateTime;
+
+pub fn datetime_to_iso_string(datetime: &DateTime) -> String {
+    format!(
+        "{:04}-{:0>2}-{:0>2}T{:0>2}:{:0>2}:{:0>2}",
```

Comment:
Brakuje uwzględniania strefy czasowej, co dostajesz po sparsowaniu? Nie jest to czas UTC?

Reply:
No właśnie nie, bo widzisz, to jest data i czas z exifa, a exif nie uwzględnia stref czasowych. W exifie data jest zapisana jako czas, który masz ustawiony w aparacie po prostu. Zresztą w takim brzydkim formacie: `2003:08:11 16:45:32`.

Także konwertując to do ISO trochę nie wiem co zrobić i postanowiłem pominąć strefę czasową i traktować wszystkie zdjęcia jako zrobione w jednej strefie czasowej, ale nie wiem czy to dobry pomysł. Nie mam też lepszego :/

Reply:
https://www.awaresystems.be/imaging/tiff/tifftags/privateifd/exif/datetimeoriginal.html

Reply:
https://www.awaresystems.be/imaging/tiff/tifftags/privateifd/exif/datetimeoriginal.html